influxql/functions.go

```diff
@@ -216,6 +218,12 @@ func (r *FloatDifferenceReducer) Emit() []FloatPoint {
 		// Calculate the difference of successive points.
 		value := r.curr.Value - r.prev.Value
 
+		// if its non_negative_difference discard any negative value. Since
```

Comment:
Can you modify this comment to use proper capitalization and fix the typo in `makred` to `marked`?